packages/kit/src/utils/routing.js

```diff
@@ -65,7 +67,9 @@ export function parse_route_id(id) {
 										const match = param_pattern.exec(content);
 										if (!match) {
```

Comment:
Could remove this whole block on the client side I think, the manifest generation would fail on this so it never gets here on the client.

Reply:
done